enable/examples/example_application.py

```diff
@@ -0,0 +1,75 @@
+# (C) Copyright 2005-2021 Enthought, Inc., Austin, TX
```

Comment:
I just realized that these should probably be private modules i.e. `enable.examples._example_application`, `enable.examples._example_support` and `enable.examples._example_canvas`. I say that because I notice external users of these modules on GitHub - https://github.com/search?q=org%3Aenthought+example_support+extension%3Apy&type=code - which I doubt was the original intention of the devs who introduced these modules.

What do you think @aaronayres35 ?

Reply:
Yeah I agree, I will rename them on this PR